pyscitt/README.md

```diff
@@ -1,18 +1,7 @@
-CCF SCITT CLI
-----------------
+# pyscitt: Python CLI tools for SCITT CCF Ledger
 
-The CLI is extensively used in functional tests and in demo scripts:
+Tools to sign claims and interact with a SCITT CCF Ledger.
 
-- [Transparency service demo](../demo/cts_poc/README.md)
-- [GitHub hosted DID demo](../demo/github/README.md)
-- [CLI tests](../test/test_cli.py)
+For more information, please find the `scitt-ccf-ledger` repository at https://github.com/microsoft/scitt-ccf-ledger.
 
-## Installation
```

Comment:
The instructions have to go somewhere, otherwise it will be confusing to anyone who lands in this repository

Reply:
Well, I removed these instructions because they do not make much sense once we merge this PR: they just provide steps on how to install pyscitt from the GH release, but once we publish the CLI to PyPi, users will be able to just install the package directly without the need to follow these steps.

In any case, I moved these instructions to the README for the moment (since the `pyscitt/README.md` will be used for the long package description available in PyPi) and we can refactor them at a later point.